spec/index.md

```diff
@@ -0,0 +1,58 @@
+# Fe Language Specification
+
+<div class="warning">
+  Warning: This is a work in progress document. It is incomplete and specifications aren't stable yet.
+</div>
+
+## Arithmetic Operators
+
+> **<sup>Syntax</sup>**\
+> _ArithmeticExpression_ :\
+> &nbsp;&nbsp; &nbsp;&nbsp; [_Expression_] `+` [_Expression_]\
```

Comment:
```suggestion
> &nbsp;&nbsp; | [_Expression_] `+` [_Expression_]\
```

Reply:
Ah, that's on purpose. It's because the `|` is used as an `or` and since this is the first operator it doesn't have an or and just uses `&nbsp;&nbsp;` for alignment. See the original for reference https://doc.rust-lang.org/stable/reference/expressions/operator-expr.html

Reply:
I see :+1: